styles/home-page.css

```diff
@@ -0,0 +1,91 @@
+@import url("base.css");
+@import url("ecom-navbar.css");
+@import url("footer.css");
+body{
+    background-color: var(--ecomm-bg);
+    font-family: 'Roboto Slab',serif;
+}
+
+.whole-content-container{
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+.main-content-container{
+    flex-grow: 1;
+}
+
+.image-banner{
+    background-image: url("../assets/images/brand-items.png");
+    background-size: auto;
+    background-repeat: no-repeat;
+    background-position: center center;
+    height:40rem;
+    display: flex;
+    width: 100%;
+}
+
+.banner-content{
+    display: flex;
+    flex-flow:column wrap;
+}
+
+.banner-title{
+    font-family: 'Amatic SC',cursive;
+    font-weight: var(--font-bold);
+    color: var(--ecomm-txt);
+}
+
+.ecom-sections{
+    background-color: var(--ecomm-secondary);
+    display: flex;
+    flex-direction: column;
```

Comment:
same as above

Reply:
Thanks @rkapoor10 for pointing to the utility classes and using it appropriately in the HTML